src/features/DisplayControls/Treatments/components/Treatments.jsx

```diff
@@ -212,13 +209,20 @@ const Treatments = (props) => {
               isAddToDrugChartDisabled ||
               moment().valueOf() <= drugOrder.effectiveStartDate
             }
-            onClick={() =>
-              handleEditAndAddToDrugChartClick(
-                drugOrder.uuid,
-                showEditDrugChartLink,
-                drugOrderSchedule?.notes
-              )
-            }
+            onClick={() => {
+              if (
+                !(
+                  isAddToDrugChartDisabled ||
+                  moment().valueOf() <= drugOrder.effectiveStartDate
+                )
```

Comment:
why do we need this condition on onClick? we are checking this condition on Link Disabled

Reply:
Yes the check was happening , the link was shaded out too . But on clicking this , the slider was still opening up

Reply:
okay